packages-next/fields-document/ARCHITECTURE.md

```diff
@@ -63,7 +63,7 @@ If you want to enforce something about the relationship between a parent and chi
 
 ## Changing the default handling of user input
 
-For some behaviour like markdown shortcuts, you need to change the default handling of user input like converting a paragraph to a blockquote when a user types `> `.
+For some behaviour like markdown shortcuts, you need to change the default handling of user input like converting a paragraph to a blockquote when a user types `>`.
```

Comment:
The space here is intentional, in the example, the blockquote is created when pressing space, not `>`

Reply:
Hmm, this is the result of a `yarn format`. Looks like we'll need to work out a way to run `yarn format` and not have it break this bit of the docs. For now I'll revert it on this branch.